core/consensus/src/wal.rs

```diff
@@ -84,7 +84,7 @@ impl SignedTxsWAL {
                 ConsensusError::Other(format!("parse folder name {:?} error {:?}", folder, err))
             })?;
 
-            if height < till {
+            if height <= till {
```

Comment:
I don't understand the variable name `till` very well.

Reply:
Till means up to the time. In this situation, means remove the transactions of the height that lower than `till`.

Reply:
Don't use this ambiguous name and consider `committed_height`.